src/Core/Grid/Definition/Factory/FeatureValueGridDefinitionFactory.php

```diff
@@ -175,6 +188,15 @@ protected function getFilters(): FilterCollectionInterface
                     ],
                 ])
                 ->setAssociatedColumn('value')
+            )
+            ->add((new Filter('position', NumberType::class))
+                ->setTypeOptions([
+                    'required' => false,
+                    'attr' => [
+                        'placeholder' => $this->trans('Search position', [], 'Admin.Actions'),
+                    ],
+                ])
+                ->setAssociatedColumn('position')
```

Comment:
```suggestion
            ->add((new Filter('position', ReorderPositionsButtonType::class))
                ->setAssociatedColumn('position')
            )
```

Reply:
We made some changes in all the grids that handle position, we removed the filter by position (which is not very useful) and instead we use this dedicated type that helps to set up the grid in a "positionable state"

A grid can only be ordered when it is sorted by position, and this custom type adds a button that helps the user understand this.